static/app/views/settings/account/notifications/notificationSettingsByProjects.tsx

```diff
@@ -72,13 +75,23 @@ class NotificationSettingsByProjects extends AsyncComponent<Props, State> {
     const canSearch = this.getProjectCount() >= MIN_PROJECTS_FOR_SEARCH;
     const shouldPaginate = projects.length >= MIN_PROJECTS_FOR_PAGINATION;
 
+    type RenderSearch = React.ComponentProps<
```

Comment:
You should pull these into a new file and use it both here and organizationMemberList